recovery.go

```diff
@@ -26,6 +27,18 @@ func NewRecovery() *Recovery {
 	}
 }
 
+// NewRecoveryWithFunc returns a nwe instance of Recovery with an
+// attached errorHandlerFunction
+func NewRecoveryWithFunc(errorHandlerFunc func(interface{})) *Recovery {
```

Comment:
Crap, I'm sorry for not including this feedback in the first place.  Can this convenience initializer be removed in favor of an addition to the README?  Something showing how to assign an error handler func to the recovery middleware...